core/tests/puppeteer-acceptance-tests/utilities/user/curriculum-admin.ts

```diff
@@ -807,6 +807,60 @@ export class CurriculumAdmin extends BaseUser {
   async openExplorationControlDropdown(): Promise<void> {
     await this.clickOn(explorationControlsSettingsDropdown);
   }
+
+  async unpublishTopic(topicName: string): Promise<void> {
+    await this.openTopicEditor(topicName);
+    await this.clickOn(' Unpublish Topic ');
```

Comment:
Are the spaces before and after the "Unpublish Topic" necessary for finding the button?

Reply:
Nope.. Can't.